### DISCUSSION TITLE
TON mining

### DISCUSSION THREAD

**Discussion by NicholasBertazzon** [Lolliedieb/lolMiner-releases#953] [General]
Hi guys,
Do someone actually know when did mining on TON started? I can't seem to find any info online. Minerstat started tracking it in december, I don't understand. Also how much TON is left to mine?

**Comment by ENTHOO** [comment]
hello everywhere
I had problems a few hours ago after mining 24, the app from one moment to another it began to throw many stale shares in a row,
solution restart lolminer app.
I check everything perfect connection, pool perfect work.
could you analyze what could be in the algorithm for "ton coin"
--ton mode 2 , pps pool "next.ton-pool.com" 
thanks to all.
